composer.json

```diff
@@ -37,5 +37,11 @@
     },
     "support": {
         "source": "https://github.com/cakephp/upgrade"
+    },
+    "require-dev": {
+        "cakephp/cakephp": "5.x-dev",
+        "cakephp/cakephp-codesniffer": "^5.0",
+        "mikey179/vfsstream": "^1.6.8",
+        "phpunit/phpunit": "^10.1"
```

Comment:
this should not be commited to prevent dependency problems for users

Reply:
Not doing a `git add -A` would also resolve the problem 😉

Reply:
You did read the part about the accident, right? :)